make-app.sh

```diff
@@ -57,12 +58,12 @@ echo "[%zuse 419]" > ${app_name}/sys.kelvin
 
 # Place the desk in given pier.
 echo "deleting old desk"
-rm -r "${path_to_pier}/${app_name}"
+rm -r "${path_to_pier}/${app_name}/*"
```

Comment:
you can't quote the *, I think this should be `"${path_to_pier}/${app_name}"/*` (at least that's what worked for me locally)